Gardening

thomasbach-dev opened the pull request:
Thank you for providing a pull request!

## Please make sure you considered the following things

- [x] I read the [contribution guidelines](./CONTRIBUTING.md).
- [x] I read the [code of conduct](./CONTRIBUTING.md#code-of-conduct).
- [ ] I created a bug report in the [Univention Bugzilla](https://forge.univention.org/bugzilla/index.cgi).
- [ ] I will add a bugzilla comment about this pull request.

## Description of the changes

A couple of things that could be cleaned up, which I found while I read the sources.

* **What kind of change does this PR introduce?** (Bugfix, feature, documentation update, cleanup, security issue, performance enhancement)

cleanup

* **Are there any breaking or API changes?** (if so, please describe them)

Nope.

* **Are there changes in the documentation necessary?**

Nope

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/univention/univention-corporate-server?pullRequest=18) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/univention/univention-corporate-server?pullRequest=18) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/univention/univention-corporate-server?pullRequest=18) it.</sub>

thomasbach-dev committed:
Make flow control better visibil.

The additional else-branch is not necessary as we already `continue`
in the if-branch. This makes the code easier to read in my opinion.

thomasbach-dev committed:
Fix typo.

reqa commented on base/univention-config-registry/python/univention-config-registry:
```diff
@@ -54,10 +54,7 @@ def convertToUtf8(arguments):
 		except UnicodeError:
-			try:
-				argument = argument.decode(localeencoding).encode('UTF-8')
-			except UnicodeError:
```
I don't thinks so. "argument" is set right from the start, otherwise argument.decode() would throw a NameError.

thomasbach-dev replied:
Of cours, you are right. Sorry for the noise. I have no idea why I oversaw this. I dropped the commit.

reqa commented on base/univention-config-registry/python/univention/config_registry/frontend.py:
```diff
@@ -140,13 +140,14 @@ def handler_set(args, opts=dict(), quiet=False):
+                                sep = sep_def
+                        else:
+                                sep = min(sep_set, sep_def)
+
```
Ok, but then indentation looks wrong, the code block must not end up being part of the if block (especially after the continue).

thomasbach-dev replied:
Sorry again. My editor doesn't like tabs. I fixed this.

reqa reviewed (changes requested):
Thanks, I've added two comments that should be addressed before this pull request is merged.

reqa commented:
Thanks for the patches, the fixes for the univention-directory-notifier are committed now into our gitlab and will sync back to github soonish. They will be compiled into the next Errata-Update for univention-directory-notifier. I'll mention your contribution in the advisory, assuming that's ok for you.

thomasbach-dev commented:
Cool. I close this pull request then.

spaceone commented:
@thomasbach-dev Can you rebase your branch? I think not all changes have been taken.

thomasbach-dev commented:
I just rebased and pushed again – looks like only the whitespace-fixes were incorporated…

spaceone commented:
I took over the typo in the function name in 8651373ff2f3ec24ca27cebaf29641fafd6cd2b7.
Only the else-removal in 4fd3ca276c52e62d05804bb3f20d949c30c323f3.

Thanks.
